### PR TITLE
Add tournament drawings wiki page.

### PR DESCRIPTION
**Pull request by smoogipooo**

### CODE REVIEW

File: wiki/Tournament_Drawings/en.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+| Property | Description | Valid Values | Default Value |
+| -------- | ----------- | ------------ | ------------- |
+| Groups | The number of groups to sort teams into. | Between 1 and 8 (inclusive) | 8 |
+| TeamsPerGroup | The maximum number of teams in any one group. | Between 1 and 8 (inclusive) | 8 |
+
+### The `drawings.txt` file
+
+This file provides a way to specify the teams that will be sorted into groups. We have already written a line into this file, so let's dissect it:
+```
+       AU       :       Australia       :       AUS
```
Comment (Repflez) [inline wiki/Tournament_Drawings/en.md]: Can this be made a table?

**Comment by smoogipooo** [comment]
Updated as per feedback received. Let me know how it looks and if all good press that green buton :P

File: wiki/Tournament_Drawings/en.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## Accessing the client
+
+Open osu!lazer and press `Ctrl + O` to reveal the game settings. From there, click "Open osu! folder". Create the following two files in this directory:  
+`drawings.ini`  
+`drawings.txt`  
+Next, add the following line to the `drawings.txt` file:  
+`AU : Australia : AUS`
+
+We'll go over what each of those files do later, however for now let's jump into the screen.
```
Comment (sr229) [inline wiki/Tournament_Drawings/en.md]: L15: Can we make this in a more neutral voice?

**Comment by sr229** [comment]
This is ready to merge @Repflez

**Pull request merged**